### PR TITLE
Feature/jobsdb header

### PR DESCRIPTION
**Pull request by manthachan**

### CODE REVIEW

**Comment by nicolassenechal** [comment]
i have no problem (there is just a lot)

How would I go at testing this? What should I expect to see?

File: jobStreet/Header/Header.demo.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+          label: 'No User Name',
+          transformProps: props => ({
+            ...props,
+            userName: ''
+          })
+        }
+      ]
+    },
+    {
+      label: 'Active Tab',
```
Comment (mengtzu) [inline jobStreet/Header/Header.demo.js]: We should probably pull out the irrelevant SEEK stuff here

File: jobStreet/Header/Header.demo.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+const ROUTE = '/header';
+
+export default {
+  route: ROUTE,
+  tenantPath: 'jobStreet',
+  title: 'Header',
+  component: Header,
+  initialProps: {
+    authenticationStatus: 'authenticated',
```
Comment (mengtzu) [inline jobStreet/Header/Header.demo.js]: Is the stuff we'll need to pass in for authentication the same among the three brands?

File: jobStreet/index.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+export { default as StyleGuideProvider } from '../react/StyleGuideProvider/StyleGuideProvider';
+export { default as SeekApp } from '../react/StyleGuideProvider/StyleGuideProvider';  // Alias for backwards compatibility
```
Comment (mengtzu) [inline jobStreet/index.js]: We won't need `SeekApp` here I don't think

File: jobsDB/Header/Header.demo.js [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+          label: 'No User Name',
+          transformProps: props => ({
+            ...props,
+            userName: ''
+          })
+        }
+      ]
+    },
+    {
+      label: 'Active Tab',
```
Comment (mengtzu) [inline jobsDB/Header/Header.demo.js]: As per the other, I think it'd be best to cut out all the SEEK specific stuff now.

Comment (manthachan) [reply jobsDB/Header/Header.demo.js]: ok, i will remove the seek stuff

File: jobsDB/Header/links.js [hunk 5]
```diff
@@ -0,0 +1,90 @@
+const isObject = obj => obj !== null
+    && !Array.isArray(obj)
+    && obj === Object(obj);
+
+const navLinks = [
```
Comment (mengtzu) [inline jobsDB/Header/links.js]: It looks like we're ending up with different links between desktop and the mobile hamburger, is that correct behaviour for jobDB?

File: jobStreet/Logo/Logo.js [hunk 6]
```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import styles from './Logo.less';
+import LogoIcon from './LogoIcon';
+
+const viewportDimensions = type => ({
```
Comment (mengtzu) [inline jobStreet/Logo/Logo.js]: Might be worth getting rid of this for our brands, it's kind of confusing if there's only one size

**Comment by mengtzu** [comment]
This is good!  I particularly like the demo stuff.  There's still a lot of refactoring I'd like us to do on the headers, but this brings it up to the same level as the JobStreet header (and actually improves that one too, thanks ^_^).

Some questions it raises though - if brand specific components are only visible if started in the appropriate tenant, do we need to make sure all three are deployed to gh pages?  (very likely).  Do we need to standardise on css-in-js for all branded content to avoid duplicating between js and less? (probably)

File: jobsDB/Header/links.js (hunk 5 above)
Comment (derekchan-seekasia) [reply jobsDB/Header/links.js]: In current jobsdb, desktop menu is different from mobile one, and I think we may need that support at the moment prior to have single navigation between both. Please check with UX

**Comment by mengtzu** [comment]
I'm going to merge this, since @manthachan is on training and I need it :)  We will need to move on improving both this and the JobStreet implementation post haste, but it's good enough for our current needs

**Pull request merged**